spec/acceptance/firewall_time_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper_acceptance'
+
+describe 'firewall type', :unless => UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+
+  describe 'reset' do
+    it 'deletes all rules' do
+      shell('iptables --flush; iptables -t nat --flush; iptables -t mangle --flush')
```

Comment:
These should go into a `before(:all)` block, as they do not actually test anything.